ml/NN/prediction_sha256_spark.py

```diff
@@ -0,0 +1,61 @@
+# This is used to predicte to the data without label. 
+# Output Example -- Sha256: 4f913360e14c5da32ff2d2f1cf1a1e788448d1d1b7eed1810da5fac4cf80aca1, Prediction: 0
+# command: 	spark-submit \
+# 			--master spark://master:7077 --py-files /FolderPath/prediction_sha256_dist.py \
+# 			--conf spark.cores.max=3 --conf spark.task.cpus=1 \
+#			--conf spark.executorEnv.JAVA_HOME="$JAVA_HOME" --conf spark.executorEnv.LD_LIBRARY_PATH="${JAVA_HOME}/jre/lib/amd64/server" \
+#			--conf spark.executorEnv.CLASSPATH="$($HADOOP_HOME/bin/hadoop classpath --glob):${CLASSPATH}" \
+#			/FolderPath/prediction_sha256_spark.py \
+#			--cluster_size 3 --input "HDFS_PARQUET_DATA" --mode inference --model HDFS_PATH_MODEL --output HDFS_PATH_OUTPUT
+
+# Tips: Code is based on the TensorflowOnSpark(TFoS) mnist example.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from pyspark.context import SparkContext
+from pyspark.conf import SparkConf
+from pyspark.sql import SQLContext
+
+import argparse
+import os
+import numpy
+import sys
+import tensorflow as tf
+import threading
+import time
+from datetime import datetime
+
+from tensorflowonspark import TFCluster
+import prediction_sha256_dist
+
+sc = SparkContext(conf=SparkConf().setAppName("NN_prediction_sha256"))
+sqlContext = SQLContext(sc)
+executors = sc._conf.get("spark.executor.instances")
+num_executors = int(executors) if executors is not None else 1
+num_ps = 1
+
+parser = argparse.ArgumentParser()
```

Comment:
This might be further in the code but if possible these should be contained in config files and not arguments. If you want both that is awesome, but other a config file is a must. Look at https://github.com/HolmesProcessing/Holmes-Analytics for an example of how to do it